doc/faq.rst

```diff
@@ -0,0 +1,155 @@
+.. _faq:
+
+.. |br| raw:: html
+
+   <div style="line-height: 0; padding: 0; margin: 0"></div>
+
+.. |hr| raw:: html
+
+   <hr/>
+
+Frequently Asked Questions
+==========================
+
+.. _q_train_deep_learning:
+
+:fa:`quora`. **Can I train a deep learning model using RSMTool?** [:ref:`#
+<q_train_deep_learning>`]
+
+``rsmtool`` supports a large number of :ref:`learners <rsmtool_model>` but not deep
+learning algorithms. However, you can still use :ref:`rsmeval <usage_rsmeval>`
+to obtain a comprehensive evaluation report from any model.
+
+|hr|
+
+.. _q_change_report:
+
+:fa:`quora`. **How can I customize** ``rsmtool`` **and** ``rsmeval``
+**reports?** [:ref:`# <q_change_report>`]
+
+The reports generated by ``rsmtool``  and ``rsmeval`` are designed to be fully
+customizable. You can:
+
+- Omit sections using the :ref:`general_sections <general_sections_rsmtool>`
+  configuration option;
+
+- Change the order of the sections using the :ref:`section_order
+  <rsmtool_section_order>` option;
+
+- Create your own :ref:`"custom" sections <custom_notebooks>`.
+
+|hr|
+
+.. _q_no_response_remaining:
+
+:fa:`quora`. **I get the following error:** ``No responses remaining after filtering out
+non-numeric feature values. No further analysis can be run.`` **What
+happened?** [:ref:`# <q_no_response_remaining>`]
+
+``rsmtool`` is designed to work only with numeric features. Non-numeric values
+*including* missing values are filtered out. Some of the common reasons for this
+error are:
+
+- The human score column or one of the features contains only non-numeric
+  values. You can either exclude this feature or convert it to one-hot
+  encoding.
```

Comment:
add a reference link to one-hot encoding? wiki page or something better if you have.